src/main/java/Gel.java

```diff
@@ -0,0 +1,56 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+import gel.Parser;
+import gel.Storage;
+import gel.TaskList;
+import gel.Ui;
+
+/**
+ * Gel is an task planner. A <code>Gel</code> object is a bot which
+ * enables user to plan their tasks.
+ */
+public class Gel {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructor for class Gel.
+     *
+     * @param filePath takes in the filepath of the storage file.
+     */
+    public Gel (String filePath) {
```

Comment:
Should this method be private since it is only used within the class? I noticed the same issue in several other places too.